english/covid-19/0/0-CC.md

```diff
@@ -1,7 +1,7 @@
-You took a painful decision. You spent the rest of your life inside the underground. The first few days was OK, but then you became very much hungry and thirsty.
+You took a painful decision. You spent the rest of your life inside the underground. The first few days was OK, but then you became very much hungry and thirsty 💀.
 
-On day 5, you are dead!
+On day 5, you are dead 🪦!
```

Comment:
What is this sign mean?

Reply:
This is an R.I.P. grave rock. Typically seen in graveyards.

Reply:
⚰ do you mean this one? because I'm seeing a sign with question mark

Reply:
Sure it is. I can see that emoji as a mac user but didn't consider non-mac users who couldn't see it. Just change it.